File: web/src/utils/isAlphaNumeric.ts
export function isAlpha(code: number): boolean {
	return (code > 64 && code < 91) || (code > 96 && code < 123);
}

export function isNumeric(code: number): boolean {
	return code > 47 && code < 58;
}

export function isAlphaNumeric(code: number): boolean {
	return isAlpha(code) || isNumeric(code);
}

File: swift/Sources/Allmark/Utils/IsAlphaNumeric.swift
@inlinable func isAlpha(code: Int) -> Bool {
	return (code > 64 && code < 91) || (code > 96 && code < 123)
}

@inlinable func isNumeric(code: Int) -> Bool {
	return code > 47 && code < 58
}

@inlinable func isAlphaNumeric(code: Int) -> Bool {
	return isAlpha(code: code) || isNumeric(code: code)
}
